resources/assets/styles/forms.less

```diff
@@ -36,9 +36,29 @@
   }
 }
 
+.track-editor {
+  .row [class*=col]:first-child {
+    margin-left: 0;
+  }
+
+  .row {
+    margin: 0;
+  }
+
+  .row .col-sm-12 {
+    padding: 0;
+  }
+
+  .row .col-sm-6 {
+    padding: 0;
+    width: 48.93617021%;
+    margin-left: 2.12765957%;
```

Comment:
I need to find out where they came from I copied them from somewhere in the app using dev tools. I'll try to get the formula instead